kclvm/ast/src/ast.rs

```diff
@@ -428,6 +428,7 @@ pub struct ImportStmt {
     pub rawpath: String,
     pub name: String,
     pub asname: Option<String>,
+    pub pkg_name: String,
```

Comment:
What does this attribute mean?

Reply:
`pkg_name` means the name of the package that the current import statement indexs to. If the current import statement indexs to the current package, `pkg_name`  is "\__main\__",  If the current import statement indexs to the external packages, `pkg_name` is the name of the package.